src/timepicker/timepicker.ts

```diff
@@ -82,8 +83,8 @@ const NGB_TIMEPICKER_VALUE_ACCESSOR = {
           <button type="button" class="btn btn-outline-primary" [class.btn-sm]="isSmallSize" [class.btn-lg]="isLargeSize"
             [disabled]="disabled" [class.disabled]="disabled"
                   (click)="toggleMeridian()">
-            <ng-container *ngIf="model?.hour >= 12; else am" i18n="@@ngb.timepicker.PM">PM</ng-container>
-            <ng-template #am i18n="@@ngb.timepicker.AM">AM</ng-template>
+            <ng-container *ngIf="model?.hour >= 12; else am">{{ i18n.getAfternoonPeriod() }}</ng-container>
```

Comment:
I would have just replaced the `"PM"` with `{{i18n.getAfternoonPeriod()}}`.

Which lead us to one question: shouldn't we keep that extra level of translation via Angular i18n to let user use it, if they would need to ?

like

```html
<ng-container
  *ngIf="model?.hour >= 12; else am"
  i18n="@@ngb.timepicker.PM">{{ i18n.getAfternoonPeriod() }}</ng-container>
```

Default content would come from Angular Locale Data, If they want to override it with a custom translation, they can.

I know that in your example, you mention that they could still use a custom implementation and provide it, but translating something outside of a template is now quite cumbersome.

Reply:
OK. I just followed what the `Datepicker` component does and used only the Angular locale data, but if you think having the ability to translate via i18n is worth it, then I'll add those translation IDs back in.

I guess leaving the translation IDs there would also mean this isn't a breaking change.

I'll make the changes and push them up tomorrow. 😄

Reply:
Done! 👍